_posts/2020-11-03-PC-Engines-Firmware-v4-12-0-7.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.12.0.7"
+date:   2020-11-27
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.12.0.7
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** 957a363.
```

Comment:
See my comment: https://github.com/pcengines/coreboot/pull/448#discussion_r528591972
and add another bulletpoint